src/app/merge-diff/conflict-editor/conflict-editor-modal/conflict-editor-modal.component.html

```diff
@@ -0,0 +1,103 @@
+<!--
+Copyright 2020-2021 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<div class="mdm-conflict-editor">
+  <div class="modal-header pxy-2">
+    <h4 class="modal-title marginless">
+      Conflict Editor
+      <i
+        class="fas fa-info-circle helpIcon"
+        (click)="loadHelp()"
+        matTooltipPosition="below"
+        matTooltip="Help"
+      ></i>
+    </h4>
+    Path: {{ data.item.path }}
+  </div>
+  <div class="modal-body pxy-2">
+    <mdm-string-conflict-editor
+      [source]="data.source"
+      [target]="data.target"
+      [item]="data.item"
+    ></mdm-string-conflict-editor>
+  </div>
+  <div *ngIf="state === 'working'" class="modal-footer pxy-2">
+    <button
+      mat-button
+      color="warn"
+      class="mr-1"
+      type="button"
+      (click)="confirmCancel()"
+    >
+      Cancel
+    </button>
+    <button
+      mat-flat-button
+      color="primary"
+      type="button"
+      (click)="startResolveConflict()"
+    >
+      Resolve conflict
+    </button>
+  </div>
+  <div *ngIf="state === 'confirmCancel'" class="modal-footer pxy-2">
+    <p>Are you sure you want to cancel? Any changes made will be lost.</p>
+    <button
+      mat-button
+      color="primary"
+      class="mr-1"
+      type="button"
+      (click)="cancel()"
+    >
+      Yes
+    </button>
+    <button
+      mat-button
+      color="warn"
+      class="mr-1"
+      type="button"
+      (click)="abort()"
+    >
+      No
+    </button>
+  </div>
+  <div *ngIf="state === 'confirmResolve'" class="modal-footer pxy-2">
+    <p>
+      Conflicts were still found in the content. Are you sure you want to
```

Comment:
I'm wondering if this should be the confirm dialog so it follows the reset of the application.